File: C++/First Steps In Programming - Lab and Exc/ConcatenationOfTextAndNumbers/ConcatenationOfTextAndNumbers.cpp
#include <iostream>
#include <string>

int main() {
    std::string firstName, lastName, town;
    int age;

    std::cout << "Enter first name: ";
    std::getline(std::cin, firstName);
    std::cout << "Enter last name: ";
    std::getline(std::cin, lastName);
    std::cout << "Enter age: ";
    std::cin >> age;
    std::cin.ignore(); // Ignore the newline character
    std::cout << "Enter town: ";
    std::getline(std::cin, town);

    std::cout << "You are " << firstName << " " << lastName << ", a " << age << "-years old person from " << town << "." << std::endl;

    return 0;
}

File: Java/PB with Java/FirstStepsInProgr - Lab/src/ConcatenationOfTextAndNumbers.java
import java.util.Scanner;

public class ConcatenationOfTextAndNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter first name: ");
        String firstName = scanner.nextLine();
        System.out.print("Enter last name: ");
        String lastName = scanner.nextLine();
        System.out.print("Enter age: ");
        int age = Integer.parseInt(scanner.nextLine());
        System.out.print("Enter town: ");
        String town = scanner.nextLine();

        System.out.printf("You are %s %s, a %d-years old person from %s.",
                firstName, lastName, age, town);
    }
}
